Feature public feed

mcardosog opened the pull request:
Feature Public Feed updated.
Added HTML (feed.html) page.
Added JS file containing code that loads all the messages from the user.

mcardosog committed:
MessageFeedServlet created

mcardosog committed:
Step 4 completed. Data store modified with getAllMessages() method. MessageFeedServlet updated with code provided in step 4

mcardosog committed:
New line added at the end of the file

mcardosog committed:
Update MessageFeedServlet.java

mcardosog committed:
feed.html added. feed-loader.js added. Server response to http://localhost:8080/feed.html properly

mcardosog committed:
feed.html added. feed-loader.js added. Server response to http://localhost:8080/feed.html properly

mcardosog committed:
Merge branch 'feature_PublicFeed' of https://github.com/ctrokid/CodeUTeam19 into feature_PublicFeed

mcardosog committed:
Merge branch 'develop' into feature_PublicFeed

ctrokid commented on src/main/webapp/user-page.html:
```diff
@@ -21,7 +21,7 @@
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
-    <script src="/js/user-page-loader.js"></script>
+    git c
```
Do we need this?

ctrokid replied:
I also don't know if we want to be removing the import of user-page-loader.js, do we?

mcardosog replied:
Apparently when I was trying to commit I had that line selected and I typed "git c" in the IDE instead the in the terminal 😭.
On the other hand I didn't change anything on that file. I believe that line 24 "<script src="/js/user-page-loader.js"></script>" is necessary ???

ctrokid replied:
Lol I hate when that happens, I have definitely done this before.

Yepp, it's necessary. Thanks for adding it back.

mcardosog committed:
Updated user-page.html

Accidentally removed line containing "<script src="/js/user-page-loader.js"></script>".
Line added (24)

mcardosog reviewed (commented):
Added "<script src="/js/user-page-loader.js"></script>" to user-page.html

mcardosog reviewed (commented):
Accidentally removed line containing "<script src="/js/user-page-loader.js"></script>".
Line added (24)
